acceptance/bundle/resources/dashboards/publish-failure-cleans-up-dashboard/out.dashboardrequests.terraform.txt

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
The files acceptance/bundle/resources/dashboards/publish-failure-cleans-up-dashboard/out.dashboardrequests.terraform.txt and acceptance/bundle/resources/dashboards/publish-failure-cleans-up-dashboard/out.dashboardrequests.direct-exp.txt look identical to me.

If they are not, can you split the files into common and different parts?